languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -20163,6 +20163,20 @@ though, and has since been slightly modified)-->
       <example correction="De toute façon"><marker>De toutes façons</marker></example>
       <example><marker>De toute façon</marker></example>
     </rule>
+      <rule id="A_TOUTE_A_L_HEURE" name="à tout à l'heure">
+          <pattern>
+              <token>à</token>
+              <marker>
+                  <token>toute</token>
+              </marker>
+              <token>à</token>
+              <token>l'</token>
+              <token>heure</token>
+          </pattern>
+          <message>Voulez-vous écrire <suggestion>tout</suggestion> ?</message>
```

Comment:
Generally, we tend to avoid simple "did you mean" messages, unless the error is really obvious. I don't think the error is super-ovious to everyone in this case.
"Dans l'expression 'tout à l'heure', 'tout' est écrit sans -e final.